resources/vue/src/composables/useGetTrips.js

```diff
@@ -0,0 +1,10 @@
+export default function useGetTrips () {
+    const useGetTrips = async (url, variable) => { //TODO: вынести в тулы
```

Comment:
Naming like this should be avoided. You can also instantiate default function without a name, like so.
```suggestion
export default function() {
    const useGetTrips = async (url, variable) => { //TODO: вынести в тулы
```